### PR TITLE
Fixes some runtimes where mobs were spawned in nullspace

### PR DESCRIPTION
**Pull request by hal9000PR**
## What Does This PR Do
fixes this
```[2021-11-24T16:32:04] Runtime in unsorted.dm,1992: Simple animal being instantiated in nullspace
[2021-11-24T16:32:04]   proc name: stack trace (/datum/proc/stack_trace)
[2021-11-24T16:32:04]   usr: The viscerator () (/mob/living/simple_animal/hostile/viscerator)
[2021-11-24T16:32:04] Runtime in unsorted.dm,1992: Simple animal being instantiated in nullspace
[2021-11-24T16:32:04]   proc name: stack trace (/datum/proc/stack_trace)
[2021-11-24T16:32:04]   usr: The viscerator () (/mob/living/simple_animal/hostile/viscerator)
[2021-11-24T16:32:04] Runtime in unsorted.dm,1992: Simple animal being instantiated in nullspace
[2021-11-24T16:32:04]   proc name: stack trace (/datum/proc/stack_trace)
[2021-11-24T16:32:04]   usr: The viscerator () (/mob/living/simple_animal/hostile/viscerator)
[2021-11-24T16:32:04] Runtime in unsorted.dm,1992: Simple animal being instantiated in nullspace
[2021-11-24T16:32:04]   proc name: stack trace (/datum/proc/stack_trace)
[2021-11-24T16:32:04]   usr: The viscerator () (/mob/living/simple_animal/hostile/viscerator)
[2021-11-24T16:32:04] Runtime in unsorted.dm,1992: Simple animal being instantiated in nullspace
```

## Why It's Good For The Game
runtimes bad

## Changelog
N/A

### CODE REVIEW

File: code/game/objects/items/weapons/grenades/spawnergrenade.dm [hunk 1]
```diff
@@ -18,10 +18,8 @@
 		for(var/mob/living/carbon/C in viewers(T, null))
 			C.flash_eyes()
 
-		for(var/i=1, i<=deliveryamt, i++)
-			var/atom/movable/x = new spawner_type
-			x.admin_spawned = admin_spawned
```
Comment (hal9000PR) [inline code/game/objects/items/weapons/grenades/spawnergrenade.dm:23]: I removed this because- it didn't really make much sense. its not admin spawned?

Comment (dearmochi) [reply code/game/objects/items/weapons/grenades/spawnergrenade.dm:23]: Presumably it takes the grenade's `admin_spawned` variable, so I would keep that in

**Review by dearmochi — CHANGES_REQUESTED** [review]

File: code/game/objects/items/weapons/grenades/spawnergrenade.dm (hunk 1 above)
Comment (hal9000PR) [reply code/game/objects/items/weapons/grenades/spawnergrenade.dm:23]: Ohhh that makes more sense now

Comment (hal9000PR) [reply code/game/objects/items/weapons/grenades/spawnergrenade.dm:23]: reverted

**Review by dearmochi — APPROVED** [review]

**Pull request merged**